website/frontend/src/components/blocks/Header/children/Notifications/Notifications.module.scss

```diff
@@ -0,0 +1,85 @@
+.notifications {
+  color: inherit;
+  position: relative;
+}
+
+.icon {
+  &.active {
+    animation: wiggle 5s 0.7s ease-in-out;
+  }
+}
+
+@keyframes wiggle {
+  0% {
```

Comment:
I think it could be implemented in the loop

Reply:
Got rid of it for now since notifications/alerts feature isn't ready yet.